developer/cmdlet/activity-parameters.md

```diff
@@ -13,215 +13,50 @@ caps.latest.revision: 5
+|**Notify**<br>Data type: SwitchParameter|Implement this parameter so that the user will be notified that the activity is complete when the parameter is specified.|
+|**NotifyAddress**<br>Data type: Email address|Implement this parameter so that the user can specify the e-mail address to use to send a notification when the **Notify** parameter is specified.|
+|**Overwrite**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet overwrites any existing data when the parameter is specified.|
+|**Prompt**<br>Data type: String|Implement this parameter so that the user can specify a prompt for the cmdlet.|
+|**Quiet**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet suppresses user feedback during its actions when the parameter is specified.|
+|**Recurse**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet recursively performs its actions on resources when the parameter is specified.|
+|**Repair**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will attempt to correct something from a broken state when the parameter is specified.|
+|**RepairString**<br>Data type: String|Implement this parameter so that the user can specify a string to use when the **Repair** parameter is specified.|
+|**Retry**<br>Data type: Int32|Implement this parameter so the user can specify the number of times the cmdlet will attempt an action.|
+|**Select**<br>Data type: Keyword array|Implement this parameter so that the user can specify an array of the types of items.|
+|**Stream**<br>Data type: SwitchParameter|Implement this parameter so the user can stream multiple output objects through the pipeline when the parameter is specified.|
+|**Strict**<br>Data type: SwitchParameter|Implement this parameter so that all errors are handled as terminating errors when the parameter is specified.|
+|**TempLocation**<br>Data type: String|Implement this parameter so the user can specify the location of temporary data that is used during the operation of the cmdlet.|
+|**Timeout**<br>Data type: Int32|Implement this parameter so that the user can specify the timeout interval (in milliseconds).|
+|**Truncate**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will truncate its actions when the parameter is specified. If the parameter is not specified, the cmdlet performs another action.|
+|**Verify**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will test to determine whether an action has occurred when the parameter is specified.|
+|**Wait**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will wait for user input before continuing when the|parameter is specified.
```

Comment:
There's an extra `|` in the line causing the table to have a 3rd column and some content in that column.

Reply:
Fixed